src/functionaltests/java/com/ericsson/ei/restendpoints/RestEndpointsTestSteps.java

```diff
@@ -38,11 +38,11 @@
     @Value("${aggregated.collection.name}")
     private String aggrCollectionName;
 
-    @Value("${missedNotificationCollectionName}")
-    private String missedNotificationCollectionName;
+    @Value("${failedNotificationCollectionName}")
+    private String failedNotificationCollectionName;
 
-    @Value("${missedNotificationDataBaseName}")
-    private String missedNotificationDatabaseName;
+    @Value("${failedNotificationDataBaseName}")
```

Comment:
I second that opinion :)

Reply:
changed to failed.notification.database-name

Reply:
You're talking about issue #337 I believe. True there are a lot of properties which needs updates, but whenever we change one, we probably need to change in a lot of places (both in code and documentation) so I think it's good to start with these 2 properties in this PR.

Reply:
@tobiasake I do agree that we should include the change  of the properties in the next release though, since it will be non backwards compatible considering all the changes in the REST API :)

Reply:
If this get prioritized in near future, then this review/comments thread can be resolved.